samples/powerpoint/shapes/get-set-shapes.yaml

```diff
@@ -0,0 +1,223 @@
+order: 2
+id: powerpoint-shapes-get-set-shapes
+name: 'Get, set, load, and save shapes'
+description: Get and set one or more selected shapes. Load and save one or more shapes.
+host: POWERPOINT
+api_set:
+    PowerPointApi: '1.5'
+script:
+    content: |
+        $("#getSelectedShapes").click(() => tryCatch(getSelectedShapes));
+        $("#setSelectedShapes").click(() => tryCatch(setSelectedShapes));
+        $("#changeFill").click(() => tryCatch(changeFill));
+        $("#saveShapeSelection").click(() => tryCatch(saveShapeSelection));
+        $("#loadShapeSelection").click(() => tryCatch(loadShapeSelection));
+        $("#createShapes").click(() => tryCatch(createShapes));
+        $("#arrangeSelected").click(() => tryCatch(arrangeSelected));
+
+        async function getSelectedShapes() {
+          await PowerPoint.run(async (context) => {
+            let finalTable = "";
+            const shapes = context.presentation.getSelectedShapes();
+            const shapeCount = shapes.getCount();
+            await context.sync();
+            finalTable += "<br>getSelectedShapes.getCount returned:<b>" + shapeCount.value + "</b><br>";
+            finalTable +=
+              "<br><table border=1 cellpadding=3 cellspacing=0><tr><td bgcolor=#3333EE><font color=white>Index</font></td><td bgcolor=#3333EE><font color=white>Id</font></td></tr>";
+            for (let i = 0; i < shapeCount.value; i++) {
+              const thisItem = shapes.getItemAt(i);
+              await context.sync();
+              thisItem.load("id");
+              await context.sync();
```

Comment:
Any plans to update so `await context.sync` statements no longer in a loop?

Reply:
Okay, should be fixed now.